### PR TITLE
Added the Arduino Wiring Porting and Issues guide

### PR DESCRIPTION
**Pull request by turkycat**

### CODE REVIEW

File: en-US/win10/ArduinoWiringPortingGuide.md [hunk 1]
```diff
@@ -0,0 +1,356 @@
+---
+layout: default
+title: Project Setup
+permalink: /en-US/win10/ArduinoWiringPortingGuide.htm
+lang: en-US
+---
+
+# Arduino Wiring Porting Guide
+
+Arduino Wiring sketches and libraries can be copy/pasted into an Arduino Wiring project inside Visual Studio and ran on Raspberry Pi 2 or Minnowboard Max. Sometimes there are slight modifications that need to be made to these files in order to make them more compatable with the Windows environment, or the board you are working with. This guide will cover those modifications as well as common issues that you may run into when deploying Arduino Wiring projects!
```
Comment (apdutta) [inline en-US/win10/ArduinoWiringPortingGuide.md]: Typo- make them more "compatible"

File: en-US/win10/ArduinoWiringPortingGuide.md [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+pinMode( GPIO_5, OUTPUT );
+digitalWrite( GPIO_5, HIGH );
+
+{% endhighlight %}
+
+The pre-defined pin names can be found in `PinNumbers.h` inside any Arduino Wiring project, but since there will be different physical connector pins available depending on the hardware setup you are building for, we've also included a table here to describe which pin names are available for each device.
+
+#### Raspberry Pi 2
+
+[Pinout Diagram]({{site.baseurl}}/images/PinMappings/RP2_Pinout.png)
```
Comment (apdutta) [inline en-US/win10/ArduinoWiringPortingGuide.md:61]: Wouldn't it be better to just have the image here rather than a link to the image?

**Comment by ooeygui** [comment]
:+1:

File: en-US/win10/ArduinoWiringPortingGuide.md [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+| SPI_CS1 | 26 |
+| SPI_CLK | 23 |
+| SPI_MISO | 21 |
+| SPI_MOSI | 19 |
+| RXD0 | 10 |
+| TXD0 | 8 |
+
+#### Minnowboard Max
+
+[Pinout Diagram]({{site.baseurl}}/images/PinMappings/MBM_Pinout.png)
```
Comment (apdutta) [inline en-US/win10/ArduinoWiringPortingGuide.md:82]: Same comment as above

File: en-US/win10/ArduinoWiringPortingGuide.md (hunk 2 above)
Comment (turkycat) [inline en-US/win10/ArduinoWiringPortingGuide.md:61]: I felt like it bloated the page with it directly embedded. What do you think?

Comment (apdutta) [inline en-US/win10/ArduinoWiringPortingGuide.md:61]: I guess that's true, although it can be a little painful to switch between pages to compare pin numbers.
That said, if the image is large enough that adding it still doesn't let you see the table and the image at the same time (i.e. you have to scroll a back and forth), it makes sense to just have the link.

**Comment by turkycat** [comment]
corrections made

File: en-US/win10/ArduinoWiringPortingGuide.md [hunk 4]
```diff
@@ -0,0 +1,356 @@
...
+| Serial.begin( int )  | *remove* |
+| Serial.write( char* str )     | *remove* *see below     |
+| Serial.print( char* str ) | Log( str )     |
+| Serial.print( int num ) | Log( num.ToString()->Begin() )      |
+| Serial.print( int num, format fmt ) | Log( num.ToString()->Begin() )      |
+
+
+####Why remove Serial.write()?
+
+Serial.write() is typically used to send raw data over the serial lines. Windows IoT Core does not currently have UART functionality (don't worry, it's coming soon!) so these types of calls should be avoided.
```
Comment (mwmckee) [inline en-US/win10/ArduinoWiringPortingGuide.md]: Are we sure we want to say "it's coming soon!" here? Feel like it might be good to avoid, in case something happens and that commitment falls through.

File: en-US/win10/ArduinoWiringPortingGuide.md [hunk 5]
```diff
@@ -0,0 +1,356 @@
...
+    myFunction();
+}
+
+{% endhighlight %}
+
+<a name="prob_hanging"></a>
+
+### My solution hangs infinitely when being initialized
+
+There is a known issue which can cause a C++ solution to hang infinitely (deadlock) when being initialized. If you find that your solution appears to have this behavior and you are unable to use the debugger to 'break in' to any statement in the setup() or loop() sections of your Arduino Wiring application, you may be experiencing this type of hanging issue.
```
Comment (mwmckee) [inline en-US/win10/ArduinoWiringPortingGuide.md]: if "your solution appears to have this behavior, ... you may be experiencing this" seems a bit wordy/redundant.

File: en-US/win10/ArduinoWiringPortingGuide.md (hunk 4 above)
Comment (turkycat) [inline en-US/win10/ArduinoWiringPortingGuide.md]: removed the comment

File: en-US/win10/ArduinoWiringPortingGuide.md (hunk 5 above)
Comment (turkycat) [inline en-US/win10/ArduinoWiringPortingGuide.md]: changed the sentence structure to remove the redundancy.

**Comment by mwmckee** [comment]
signoff

**Comment by anthony-ngu** [comment]
Merged with Matt, Apu, and Lou's signoff
